src/main/java/figures/Main.java

```diff
@@ -0,0 +1,29 @@
+package figures;
+
+public class Main {
+    public static void main(String[] args) {
+        int size = (int) (Math.random() * 100);
+        Shape[] shapes = new Shape[size];
+
+        for (int i = 0; i < size; i++) {
+            int random = (int) (Math.random() * 4);
```

Comment:
let's make 4 a constant so it would clear why it's 4 and not some other number
FIGURES_TYPES_AMOUNT